docs/specs/tripgo.swagger.yaml

```diff
@@ -2161,6 +2161,9 @@ definitions:
       batteryLevel:
         description: Battery level of electric vehicles, ranging from 0-100
         type: integer
+      qrCode:
+        description: QR code of the vehicle
```

Comment:
@MarianoTucat On second look, I'm confused about this. What does it mean that a vehicle has a QR code? What would you use it for?

Reply:
This is related to https://redmine.buzzhives.com/issues/12985, agree that qrCode is confusing. For neuron, it will we the IMEI, not a qrCode, maybe, we should change the name to "externalID"? or something similar.

Reply:
In that case, it should be called IMEI.